QuestPatcher.QMod/QModManifest.cs

```diff
@@ -68,13 +68,16 @@ public string VersionString
         
         /// <summary>
         /// The package ID of the app that the mod is designed for.
+        /// If null, this means that the mod works for any app.
         /// </summary>
-        public string PackageId { get; set; }
+        public string? PackageId { get; set; }
 
         /// <summary>
         /// The version of the app that the mod is designed for.
+        /// If null, this means that the mod doesn't depend on a particular version of an app.
+        /// Specifying this is redundant if <see cref="PackageId"/> is specified.
```

Comment:
What's the verbage meant to mean here? This makes it seem like specifying the `packageVersion` is redundant if the `packageId` is specified. Isn't it the opposite that is the case?

Reply:
Perhaps using a word such as "omitting this is redundant..." or "specifying this is redundant... is not specified"